Sources/OutlineView/OutlineView.swift

```diff
@@ -55,27 +74,35 @@ where Data.Element: Identifiable {
     ///     at the key path is `nil`, then the outline group treats `data` as a
     ///     leaf in the tree, like a regular file in a file system.
     ///   - selection: A binding to a selected value.
+    ///   - dropReceiver: An object conforming to `DropReceiver` that handles
+    ///     drag-and-drop receiving for the OutlineView
+    ///   - separatorInsets: An optional closure that produces row separator lines
+    ///     with the given insets for each item in the outline view. If this closure
+    ///     is not provided (the default), separators are hidden.
     ///   - content: A closure that produces an `NSView` based on an
     ///     element in `data`. An `NSTableCellView` subclass is preferred.
     ///     The `NSView` should return the correct `fittingSize`
     ///     as it is used to determine the height of the cell.
+    @available(macOS 11.0, *)
     public init(
         _ data: Data,
         children: KeyPath<Data.Element, Data?>,
         selection: Binding<Data.Element?>,
+        dropReceiver: Drop,
+        separatorInsets: ((Data.Element) -> NSEdgeInsets)? = nil,
         content: @escaping (Data.Element) -> NSView
```

Comment:
That was dumb of me. Must have been using this only in earlier code testing, because once I removed the `@available(macOS 11.0, *)`, everything still built fine. Oops! https://github.com/Sameesunkaria/OutlineView/pull/10/commits/bdb22722ae865234c4ce8fa83c76bcab52a067f3

Reply:
For some historical reference: https://github.com/Sameesunkaria/OutlineView/commit/b6a3a2dffd41e5354e0ef539b2b6df0b016011be

`separatorInsets` are not supported on macOS 10.15 and the corresponding initialisers and modifiers should not be available when targeting it.

In effect, this means that all your initializers that take `separatorInsets` should be restricted to macOS 11 and up, and have a second variant that that does not take a `separatorInsets`.

Reply:
Aha! I had forgotten about that. Okay, adding those inits back in here: https://github.com/Sameesunkaria/OutlineView/pull/10/commits/5dc97ed8d7a56d29bea640d2b20c13284d90fe30

That puts us up to 8 initializers now... Not sure if that's a problem, but I'm trying to keep them organized.

Reply:
> That puts us up to 8 initializers now

That is not uncommon for native SwiftUI APIs as well. Ultimately, the ergonomics of the API matter the most. Here even though we have several initializers, they aren't adding complexity from the developers perspective.